### ISSUE TITLE
Project structure

### ISSUE THREAD

**Issue by repetti** [ltguillaume/droidshows#8]
How about changing it to standard android project structure? This will ease the import and build process.
It is Eclipse project now, but changing it to Android Studio project can be useful and allow building it without having Eclipse installed and started but only using Gradle.

**Comment by ltguillaume** [comment]
Hi, thanks for the German and Russian translations, I've merged your pull request and they'll be included in the next release.

As for Android Studio, I have tried to use it in the past, but it's simply a no-go on my hardware: Eclipse is already quite slow, but it's simply impossible for me to run Android Studio (even though it's "supposed to be faster" according to some people; I'm guessing it's just placebo for them because they prefer Android Studio, for me it's absolutely not the case). Furthermore, I think it introduces useless overhead to the project: personally I hate Gradle as it has only introduced new problems for me in the past. It's only more or less useful if the project were to have many dependencies. DroidShows has 0 dependencies, not even the Android v4 support library (I think I saw it added in the the gradle files in your fork).

I do realize that it would be much easier for other people to participate in the development of DroidShows if I were to convert the project structure, but that would simply mean that I wouldn't be able to work on it anymore, or I would have to keep the gradle files up to date manually. Furthermore it would give more overhead to the project and possible the APK.

I'll re-evaluate this if there comes a time that I buy new hardware.
